scripts/format.js

```diff
@@ -68,7 +68,16 @@ async function runOnChanged() {
   const cmd = `git --no-pager diff ${passedDiffTarget} --diff-filter=AM --name-only --stat-name-width=0`;
 
   const gitDiffOutput = execSync(cmd, { cwd: root });
-  const files = gitDiffOutput.toString('utf8').split('\n');
+  const files = gitDiffOutput
```

Comment:
applied fix to properly handle empty diffs . cc @layershifter . please re-review. thx